package.json

```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "scripts": {
     "dev": "set NODE_ENV=development&&gulp --gulpfile=./gulpfile.js",
+    "dev-fresh": "set NODE_ENV=development&&gulp --gulpfile=./gulpfile.js dev:run-fresh",
     "prod": "set NODE_ENV=production&&gulp prod --gulpfile=./gulpfile.js",
+    "prod:run": "set NODE_ENV=production&&gulp prod:run --gulpfile=./gulpfile.js",
     "build": "set NODE_ENV=development&&gulp build --gulpfile=./gulpfile.js",
     "build:prod": "set NODE_ENV=production&&gulp build --gulpfile=./gulpfile.js",
+    "build:vendor-manifest": "webpack --config=./build/webpack/webpack.vendor.config.js",
+    "rebuild": "set NODE_ENV=development&&gulp build --gulpfile=./gulpfile.js rebuild",
```

Comment:
@carterbs what's the difference between rebuild and dev-fresh? What about clean?
@carterbs what is build:vendor-manifest?
@NinjaManatee what is prod:run?